src/modules/privateChat/components/doctors-list-card.component.tsx

```diff
@@ -21,36 +21,36 @@ interface DoctorsListCardProps {
  * @param param0
  * @returns
  */
-const DoctorsListCardItem = () => {
+const DoctorsListCardItem = (doctor: any) => {
   const navigation = useNavigation() as any;
-  let user = {
-    profile_image: "https://i.pravatar.cc/200",
-    full_name: "Doctor Raji Mike",
-    role: "Pediatrician",
-    rating: 4,
-    points: 5,
-  };
+  // let user = {
+  //   profile_image: "https://i.pravatar.cc/200",
+  //   full_name: "Doctor Raji Mike",
+  //   role: "Pediatrician",
+  //   rating: 4,
+  //   points: 5,
+  // };
 
   return (
     <View style={styles.header}>
       <TouchableWithoutFeedback
         onPress={() => {
           navigation.navigate(
             NAVIGATION_CONSTANTS.SCREENS.PRIVATE_CHAT.DOCTOR_PROFILE_SCREEN,
-            { user }
+            { doctor }
           );
         }}
       >
         <View style={styles.header__avatar_and_details}>
           <Image
-            source={{ uri: user.profile_image }}
+            source={{ uri: doctor.doctor.profile_image }}
```

Comment:
@oluwadamilarey 

Read the comment above and fix this, so it's doctor.profile_image